docs/book/src/user/quick-start.md

```diff
@@ -158,6 +158,23 @@ Test to ensure the version you installed is up-to-date:
 clusterctl version
 ```
 
+{{#/tab }}
+{{#tab windows}}
+
+#### Install clusterctl binary with curl on windows
+Go to the working directory where you want clusterctl downloaded.
+
+Download the latest release; on windows, type:
+```
+curl {{#releaselink gomodule:"https://github.com/kubernetes-sigs/cluster-api/releases/download/v1.1.1/clusterctl-windows-amd64.exe"}} -o clusterctl.exe
+```
+Add the path of that directory to path environment variable.
```

Comment:
Could you put an example of how to set the path here here? Or maybe just add a directory that's normally on the Path in Windows. That's what we're doing in the Linux / MacOS guides

Reply:
Yeah I was also thinking of adding command which is being used to add directory to environment path.
But didn't find anything. Everywhere I just saw the UI steps to add directory path to environment variables.

Reply:
I am not a 100% sure - I will let you test this on a windows machine but I think you can add to the path in the current session using `set PATH=%PATH%;C:\your\path\here\` but to add it permanently you might have to use the GUI. 
If there is any standard documentation that shows how to add to the PATH using GUI then it would be nice to have that linked here.